themes/10up-theme/tests/includes/classes/ModuleInitializationTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Test Class
+ *
+ * @package TenUpTheme
+ */
+
+namespace includes\classes;
+
+/**
+ * Test Class
+ */
+class ModuleInitializationTest extends \WP_UnitTestCase {
+
+	/**
+	 * Undocumented variable
+	 *
+	 * @var [type]
+	 */
+	public $class;
+
+	/**
+	 * Set up test fixtures.
+	 *
+	 * @return void
+	 */
+	public function set_up() {
+		parent::set_up();
+
+		$this->class = \TenUpTheme\ModuleInitialization::instance();
+	}
+
+	/**
+	 * Ensure we can find the right classes.
+	 *
+	 * @return void
+	 */
+	public function test_it_can_find_classes() {
+		$this->class->init_classes();
+
+		$class  = new \ReflectionClass( $this->class );
+		$method = $class->getMethod( 'get_classes' );
+
+		$classes = $method->invoke( $this->class );
+		$this->assertCount( 2, $classes );
```

Comment:
I think this should be a count greater than 1. If it can find 1 then it's working. Otherwise you need to update this test every time you include another class.

Reply:
Good call, I had this on a project that I'm using this on. I'll update with the tests I'm using there!